README.md

```diff
@@ -15,3 +16,5 @@ the function and run automatically when you commit through a GitHub action.
 ## Runtime Analysis
 
 What is the worst-case big $\Theta$ time complexity of your algorithm?
+
+- In the worst case, we compare all vertices in graph1 to all vertices in graph2 (v*v), then in the recursive step, we check to see if that particular mapping works (v! possible mappings), thus the worst case time complexity can be bounded by $\Theta(v^2 * v!)$
```

Comment:
Don't understand how you get to this. The recursive call is on a smaller version of the problem, right? So it shouldn't need to explore $|V|!$ possibilities.